### ISSUE TITLE
blank str topics crash object detail pages

### ISSUE THREAD

**Issue by GeoffFroh** [densho/ddr-public#55]
ddr-densho-305 is returning 500 errors from the ddr-public application for all of the object details pages. See:

http://ddr.densho.org/ddr/densho/305/4

Nothing obvious in the logs (/var/log/ddr/public.log, /var/log/nginx/ddrpublic.log, /var/log/ddrpublic.err), and there doesn't seem to be anything wrong with the mezzanines. See:

http://ddr.densho.org/ddr/densho/305/4/mezzanine/022edf8dab/

You can also access an individual master file detail page:

http://ddr.densho.org/ddr/densho/305/4/master/72a82616be/

One difference between this collection and the batch of others that were just indexed is that it contains master files in JPEG2000 (.j2k) format. However, the standard format jpeg access files are present (and were generated by ddr-local).

**Comment by GeoffFroh** [comment]
Sample raw json document from elasticsearch index:

[ddr-densho-305-4-esjson.txt](https://github.com/densho/ddr-public/files/477446/ddr-densho-305-4-esjson.txt)

**Comment by gjost** [comment]
Nothing to do with the graphics, it's actually a blank `str` in the entity's `topics` list:
```
ValueError at /ddr/densho/305/4/
invalid literal for int() with base 10: ''
```

**Comment by gjost** [comment]
Fixed in 237eaa2.
